### ISSUE TITLE
Module import error for documentation analysis\clustering\sax

### ISSUE THREAD

**Issue by Jack-Hawthorne** [architecture-building-systems/CityEnergyAnalyst#1723]
The following error is raised by sphinx autodoc when making the documentation:

```
WARNING: autodoc: failed to import module u'clustering_main' from module u'cea.analysis.clustering.sax'; the following exception was raised:
cannot import name SAX
WARNING: autodoc: failed to import module u'sax_optimization' from module u'cea.analysis.clustering.sax'; the following exception was raised:
cannot import name SAX
```
